README.md

```diff
@@ -1 +1,13 @@
-# sql-topic-introduction
+# Introduction To Structured Query Language
+
+## Objectives
+
+1. Understand the basic definition of SQL. 
+2. Understand why learning SQL and database management are essential skills for a web developer. 
+3. Get a brief overview of this unit. 
+
+## notes for whoever writes this:
+
+For the second objective, use the text from [this readme](https://github.com/learn-co-curriculum/why-sql-matters). 
+
+For the third objective, outline the basic flow/objectives of the unit. Something like: "by the end of this unit you will know how to create SQLite3 databases, create database tables, insert data into, update the data within, select and delete data from those database tables. You will learn how data within a given database can be related and how to write Ruby programs that talk to your databases and actually save data into them."
```

Comment:
So I changed my mind and think it might be good to define SQL in the topic intro, so I took that from the subsequent readme and added it here. I also don't think we need to have objectives listed at the top for topic intros since this is more to just get people acquainted with what they're about to cover. Instead, I added the objectives you had Sophie as bullet points at the end. 

As a full-stack web developer, you'll frequently be working with databases to manage the data associated with your applications. Think about the web apps you're familiar with. Facebook saves user data and stores -- or  persists -- your associations to your friends. Amazon has an enormous database of items for sale. The New York Times has a storage system for all of their articles. Most of the sites you interact with every day, and consequently most of the sites you'll work on or build, need databases to persist data. 

We've already seen how we can build Ruby programs that model real-world objects and environments. With database management skills, we'll learn to store representations of the Ruby objects our programs create and retrieve them at the appropriate time. We’ll also learn how to connect our Ruby (and even later, Ruby on Rails) applications to our databases. For example, a basic web application might have many users. So far, we've learned how to build a Ruby User class that produces user objects. But, we don't yet know how to store those users and their details. If a user signs up for our app and then we lose all their information immediately, then how for example, will we know if an existing user is signing back into our app. We need a way to take our Ruby objects, store them in a database and retrieve them at the appropriate time.

This topic will cover how to use and navigate databases based on SQL. SQL (Structured Query Language) is a language for managing data in a database. Unlike some other sorts of programming languages, it's only used for one thing: talking to databases. Thus, you might hear it referred to as a "special purpose" programming language, which means that you won't be using SQL to write the next big web app, but you might use it to interact with the database that powers it. 

Even though SQL really only has one purpose, it is used by many different database systems such as MySQL, PostgreSQL, or the system we'll being using in this course: SQLite. 

By the end of this topic, you’ll be able to: 
Create SQLite3 databases
Create, update, select, and delete data from database tables
Relate data within a given database
Write Ruby programs that talk to and save data to your databases